src/CoreWf/IAsyncTaskCodeActivity.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Activities
+{
+    internal interface IAsyncTaskCodeActivity : IAsyncCodeActivity
```

Comment:
These interfaces are only used in one place. I added them for consistency with the `AsyncCodeActivity` type, however they could easily be done away with if we feel it adds bloat.

Reply:
Yes, I don't think we need these interfaces.